### ISSUE TITLE
missing parentage to 'cell junction'

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#18768]
1. Is there any reason why 'cell junction' (A cellular component that forms a specialized region of connection between two or more cells, or between a cell and the extracellular matrix, or between two membrane-bound components of a cell, such as flagella")

is not a parent to 
synapse?

2. the def for synapse says 'The junction between a nerve fiber of one neuron and another neuron...'
but nerve fiber is not a GO term - change to axon + add nerve fiber as a synonym to the term axon for searching?

**Comment by ukemi** [comment]
@BarbaraCzub didn't you work on this?

**Comment by BarbaraCzub** [comment]
Hi @ukemi 
I have definitely worked on 'adherens junction' (e.g. https://github.com/geneontology/go-ontology/issues/18119) and this term is a descendant of cell-junction:
![image](https://user-images.githubusercontent.com/16541096/73942182-af1d1300-48ff-11ea-995c-e4be8f7188a1.png)

'Gap junction' is also looking fine:
![image](https://user-images.githubusercontent.com/16541096/73942280-d2e05900-48ff-11ea-9593-a4250f8811c5.png)

Wrt to synapse, I am happy to add the 'cell junction' parent here unless there could have been a reason why the SynGO project did not do this?

**Comment by ukemi** [comment]
I generally don't think of a synapse as a cell junction because there is no physical contact between the two cells, but am happy to go with the opinion of the experts.

**Comment by Antonialock** [comment]
@BarbaraCzub thanks, oops, not sure why I didn't see that when I was looking at the terms

**Comment by BarbaraCzub** [comment]
> I generally don't think of a synapse as a cell junction because there is no physical contact between the two cells, but am happy to go with the opinion of the experts.

Perhaps this is the reason why SynGO did not make synapse a child of cell junction. I am hesitant about this too.

**Comment by BarbaraCzub** [comment]
> @BarbaraCzub thanks, oops, not sure why I didn't see that when I was looking at the terms

Can be easily missed! But the parentage is definitely there for adherens and gap junctions. It's just not direct.

**Comment by Antonialock** [comment]
hen the def could just be clarified to specify 'physical connection'?

**Comment by BarbaraCzub** [comment]
Well. Actually, maybe the 'cell junction' was not intended to specify a physical connection and then it would be fine to make synapse a child of it. 

Because the definition of its child: 'anchoring junction', which is a grouping term for all the adherens and gap and related physical junctions, specifies a physical connection: "A cell junction that mechanically attaches a cell (and its cytoskeleton) to neighboring cells or to the extracellular matrix."

**Comment by ukemi** [comment]
I always thought that it was meant to specify a physical connection, but will take @BarbaraCzub's advice with respect to the synapse if she thinks the community would expect it there. @BarbaraCzub, can you make the edits?

**Comment by BarbaraCzub** [comment]
Yes, I believe many researchers think of a 'synapse' as a 'junction'. In fact, even the definition of the synapse in GO states this. So, I will make the 'synapse' a child of 'cell junction', but I will change the 'nerve fibre' in the definition to a 'neuron projection' (which includes both 'axons' and 'dendrites').

**Comment by ukemi** [comment]
We should look a the immunological synapse too. I don't know much about it, but remember it exists. @addiehl?

**Comment by BarbaraCzub** [comment]
I suppose it should also fall under 'cell junction', but I'll wait for @addiehl to comment before making this edit.

**Comment by ValWood** [comment]
@Antonialock  what is the action?
I have unassigned @BarbaraCzub  as she is no longer with GO. Can you summarize of close?

**Comment by Antonialock** [comment]
My original request has been added. It looks like it sparked another potential ticket item to potentially  add "cell junction" parentage to " immunological synapse" too, but that is not what I requested so I suppose this can close and a new ticket can be opened for other terms
